about-pses/example-pse.md

```diff
@@ -44,17 +44,36 @@ List three or more questions whose answers would clarify the problem statement.
 
 ##### !explanation 
 
-Here are some example clarifying questions:
-
-1. How should the function handle invalid user input (i.e. player_1 = "lizards")?
-2. Should a user automatically lose if they have invalid input?
-3. What should happen if both users have invalid input?
-4. Does capitalization matter?
-5. Does extra whitespace / punctuation matter?
-6. Should anything be printed to the console?
-7. Is the speed / memory usage of this function important?
-8. Should there be a default value for each of the arguments?
- 
+The following list provides some example clarifying questions, as well as what we might decide for ourselves is the answer. Several of these questions show examples of how we might think about the same issue in different ways. For example, questions 1, 2, and 3 are all about how to handle invalid input, but they each conceive of the problem slightly differently. Questions 4 and 5 are both getting at what variations in input would be considered valid or invalid. Questions 6 and 7 are about how to handle the possibility of extending the game to include more moves or more players. Notice that, even though we decide to stick with a more focused implementation for those questions, thinking through what some logical extensions might be (and in an interview context, bringing them up with the interviewer) can show that we're thinking about the problem in a broader context.
+
+<br>
+
+This prompt is pretty tightly specified, making it potentially more difficult to come up with clarifying questions. For future PSEs, there may be areas around edge cases, or sometimes even aspects of the core logic that may be more open to interpretation. Try to keep an eye out for areas of logic where it feels like you could make a decision about how to handle it, and then ask a question that would help you make that decision.
+
+<br>
+
+Try not to pose questions that are too specific to the implementation. For example, "Should I use a dictionary to store the winning combinations?" is too specific, and an interviewer is likely to leave choices about an implementation to us (and we should be prepared to justify our decisions). We should also avoid questions that change the observable behavior of a prompt. While we shouldn't ask about changing parameters or return values, or modifying behavior to something that conflicts with the prompt, we _could_ observe the requirements of the prompt, and mention how we might approach it if it were solely up to us. In an interview setting, this can give a sense of our thought process, and convey familiarity with the language over all, opening up space for a discussion. Ideally, we should think about questions that would help us understand the problem better and make decisions about how to approach it.
```

Comment:
How would you feel about moving these sections from the explanation to hints? I feel like it's information that could help folks think a little differently if they are feeling stuck, and isn't tied directly to the questions you share below.